src/ATen/CMakeLists.txt

```diff
@@ -63,30 +65,31 @@ IF ($ENV{TH_BINARY_BUILD})
   SET(CMAKE_CXX_FLAGS "-static-libstdc++ ${CMAKE_CXX_FLAGS}")
 ENDIF()
 
-IF(NO_CUDA)
-  MESSAGE(STATUS "ignoring CUDA")
+IF(NOT NO_CUDA)
+  FIND_PACKAGE(CUDA 5.5)
+ENDIF()
+
+IF(NO_CUDA OR NOT CUDA_FOUND)
+	MESSAGE(STATUS "ignoring CUDA")
```

Comment:
I was trying to make it consistent with pytorch (https://github.com/pytorch/pytorch/blob/master/tools/setup_helpers/cuda.py), which does the same thing.